app/models/course/assessment/answer.rb

```diff
@@ -107,6 +107,17 @@ def grade_inline?
     end
   end
 
+  # Whether the answer has downloadable content.
+  #
+  # @return [Boolean]
+  def downloadable?
+    if actable.self_respond_to?(:downloadable?)
```

Comment:
Yah, `respond_to` will delegate to parent ( which is answer in this case  ) , so this is necessary